smsapi/Api/Action/Contacts/CreateContact.cs

```diff
@@ -4,136 +4,126 @@
 
 namespace SMSApi.Api.Action
 {
-    public class CreateContact : Rest<Contact>
+    public class CreateContact : Base<Contact>
     {
-        public DateTime? BirthdayDate;
-
-        public string City;
-
-        public string Description;
-
-        public string Email;
-
-        public string FirstName;
-
-        public string Gender;
-
-        public string LastName;
-
-        public string PhoneNumber;
-
-        public string Source;
-
-        protected override RequestMethod Method => RequestMethod.POST;
-
-        protected override NameValueCollection Parameters
-        {
-            get
-            {
-                NameValueCollection parameters = base.Parameters;
-                if (PhoneNumber != null)
-                {
-                    parameters.Add("phone_number", PhoneNumber);
-                }
-
-                if (Email != null)
-                {
-                    parameters.Add("email", Email);
-                }
-
-                if (FirstName != null)
-                {
-                    parameters.Add("first_name", FirstName);
-                }
-
-                if (LastName != null)
-                {
-                    parameters.Add("last_name", LastName);
-                }
-
-                if (Gender != null)
-                {
-                    parameters.Add("gender", Gender);
-                }
-
-                if (BirthdayDate != null)
-                {
-                    parameters.Add("birthday_date", BirthdayDate.Value.ToString("Y-m-d"));
-                }
-
-                if (Description != null)
-                {
-                    parameters.Add("description", Description);
-                }
-
-                if (City != null)
-                {
-                    parameters.Add("city", City);
-                }
-
-                if (Source != null)
-                {
-                    parameters.Add("source", Source);
-                }
-
-                return parameters;
-            }
-        }
-
-        protected override string Resource => "contacts";
+        private DateTime? birthdayDate;
```

Comment:
You should not make these properties private, it's BC, someone might already used direct access.

Reply:
There is also an issue with that nullable fields/properties and removing nullable is also a BC. Should we just bump up version to 2.0.0 and keep it like it is right now? ;)

Reply:
Yes, I wrote this comment while checking, I wasn't prepared for so many changes :)